eknrenderer/configure.ac

```diff
@@ -0,0 +1,147 @@
+AC_SUBST([EKN_RENDERER_MINOR_VERSION], [_EKN_RENDERER_MINOR_VERSION_MACRO])
+AC_SUBST([EKN_RENDERER_MICRO_VERSION], [_EKN_RENDERER_MICRO_VERSION_MACRO])
+AC_DEFINE([EKN_RENDERER_VERSION], [_EKN_RENDERER_VERSION_MACRO], [Full version number])
+
+# Before making a release, the EKN_RENDERER_LT_VERSION_INFO string should be modified. The
+# string is of the form c:r:a. Follow these instructions sequentially:
+#   1. If the library source code has changed at all since the last update, then
+#      increment revision (‘c:r:a’ becomes ‘c:r+1:a’).
+#   2. If any interfaces have been added, removed, or changed since the last
+#      update, increment current, and set revision to 0.
+#   3. If any interfaces have been added since the last public release, then
+#      increment age.
+#   4. If any interfaces have been removed or changed since the last public
+#      release, then set age to 0.
+AC_SUBST([EKN_RENDERER_LT_VERSION_INFO],[0:0:0])
+
+# Required build tools
+# --------------------
+# Make sure we can create directory hierarchies
+AC_PROG_MKDIR_P
+# C compiler
+AC_PROG_CC
+AC_PROG_CC_C99
+# Library configuration tool
+PKG_PROG_PKG_CONFIG
+# Gtk-doc; 0.18 required for Markdown parsing
+GTK_DOC_CHECK([1.18])
+# GObject Introspection
+GOBJECT_INTROSPECTION_REQUIRE([1.30])
+# Various tools
+AC_PROG_AWK  # needed for TAP driver
+AC_PROG_SED  # needed for finding files for coverage
+AC_REQUIRE_AUX_FILE([tap-driver.sh])
+AC_PATH_PROG([GLIB_MKENUMS], [glib-mkenums], [notfound])
+AS_IF([test x$GLIB_MKENUMS = xnotfound], [AC_MSG_ERROR([glib-mkenums not found.
+Try installing the glib development package.])])
+
+AC_CACHE_SAVE
+
+# Check that the compiler really supports C99
+AS_IF([test "x$ac_cv_prog_cc_c99" = "xno"], [AC_MSG_ERROR([C99 is required.])])
+
+AX_COMPILER_FLAGS
+
+# JASMINE_JUNIT_REPORTS_DIR: Where to put test reports
+AC_MSG_CHECKING([where to put test reports])
+AC_ARG_VAR([JASMINE_JUNIT_REPORTS_DIR], [Where to put test reports])
+AS_IF([test -n "$JASMINE_JUNIT_REPORTS_DIR"],
+    [JASMINE_REPORT_ARGUMENT="--junit $JASMINE_JUNIT_REPORTS_DIR/\$\${log/%.log/.js.xml}"
+    AC_MSG_RESULT([in $JASMINE_JUNIT_REPORTS_DIR])],
+    [JASMINE_REPORT_ARGUMENT=
+    AC_MSG_RESULT([nowhere])])
+AC_SUBST([JASMINE_REPORT_ARGUMENT])
+
+# Package dependencies
+# Update these whenever you use a function that requires a certain API version
+AX_PKG_CHECK_MODULES([EKN_RENDERER], [glib-2.0 gio-2.0 gobject-2.0 json-glib-1.0 endless-0 mustache_c-1.0])
+
+# Code Coverage
+# -------------
+AX_CODE_COVERAGE
+
+# Substitutions for defines
+AC_ARG_WITH([mathjax-dir], AS_HELP_STRING([--with-mathjax-dir],
+    [Use mathjax from a location besides the default /usr/share]),
+    [MATHJAXDIR=$withval],
+    [MATHJAXDIR=/usr/share/javascript/mathjax])
+AC_MSG_CHECKING([for mathjax javsacript library])
+if test -d $MATHJAXDIR; then
+    AC_MSG_RESULT([yes])
+    AC_SUBST([MATHJAXDIR])
+else
+    AC_MSG_ERROR([Cannot find mathjax package])
+fi
```

Comment:
Having to do this again in the internal lib's `configure.ac` seems really suboptimal to me, but I couldn't figure out a way to get the substitution to propagate from the parent.